src/main/java/seedu/address/model/client/insurance/InsurancePlan.java

```diff
@@ -31,6 +31,20 @@ public int getInsurancePlanId() {
         return insurancePlanId;
     }
 
+    /**
+     * Removes a claim from the insurance plan's list of claims, if a claim with the same claim id exists.
+     *
+     * @param claim The claim to be removed.
+     */
+    public void removeClaim(Claim claim) {
+        for (Claim c : claims) {
+            if (c.getClaimId().equals(claim.getClaimId())) {
+                claims.remove(c);
+                break;
+            }
```

Comment:
Since the implementation is an Arraylist<Claims>, is it possible to just remove without the for loop?

Reply:
I was thinking about that too. I can remove the for loop and add equals method for Claim.java. However, the equals method I will be implementing will take 2 claims as equal if they have the same ClaimId only. It will not check if the claims have the same ClaimAmount and I wasn't sure if there will be any repercussions resulting from this

Reply:
Yep 👍